demos/node-classification-graphsage/graphsage-cora-example.py

```diff
@@ -99,8 +99,12 @@ def train(
         test_nodes, test_targets, train_size=500, test_size=None, random_state=523214
     )
 
+    # Create graph schema
```

Comment:
why do we need to create the graph schema and pass it to the node generator? This extra step was not there before, what is the justification for putting it in the demo, making the workflow more complicated?